src/containers/TxDetails.js

```diff
@@ -30,17 +30,17 @@ const fetchAccountName = (state, address = '') => {
 const mapStateToProps = (state, ownProps) => ({
```

Comment:
We should also display the message to be signed, and it requires a separate screen for that.